src/app/features/errors/app-error-boundary.tsx

```diff
@@ -7,9 +11,73 @@ import { Header } from '@app/components/header';
 import { ErrorBoundary, FallbackProps, useErrorHandler } from '@app/features/errors/error-boundary';
 import { openGithubIssue } from '@app/features/errors/utils';
 import { useErrorStackTraceState } from '@app/store/ui/ui.hooks';
+import { Button } from '@app/ui/components/button/button';
 import { CodeBlock } from '@app/ui/components/codeblock';
 import { Title } from '@app/ui/components/typography/title';
 
+export function RouterErrorBoundary() {
+  const error = useRouteError() as Error;
+
+  const title = 'Something went wrong';
+  const body = 'An error occurred in the app.';
+  const errorPayload = error.message;
+
+  return (
```

Comment:
Can you please refactor this so it's a dumb component, receiving some props it needs and located in the `ui/components` folder? 

We should also add a storybook story for it so we can see how it looks